src/amo/reducers/guides.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
+import invariant from 'invariant';
+
+import { createPlatformFiles } from 'core/reducers/addons';
+import type { AddonType } from 'core/types/addons';
+
+export const FETCH_GUIDES_ADDONS: 'FETCH_GUIDES_ADDONS' = 'FETCH_GUIDES_ADDONS';
+export const LOAD_GUIDES_ADDONS: 'LOAD_GUIDES_ADDONS' = 'LOAD_GUIDES_ADDONS';
+
+export type FetchGuidesParams = {|
+  errorHandlerId: string,
+  guid: string,
+|};
+
+export type FetchGuidesAction = {|
+  type: typeof FETCH_GUIDES_ADDONS,
+  payload: FetchGuidesParams,
+|};
+
+export type GuidesState = {|
+  addons: Array<AddonType> | null,
+|};
+
+export type LoadGuidesAction = {|
+  type: typeof LOAD_GUIDES_ADDONS,
+  payload: GuidesState,
+|};
+
+export const initialState: GuidesState = {
+  addons: null,
+};
+export const loadGuidesAddons = ({ addons }: GuidesState): LoadGuidesAction => {
```

Comment:
I believe this is not needed. Let's add the addons in the `addons` reducer instead.

Reply:
hey @willdurand, can you please clarify what you mean?

are you saying that the redux structure should be something like ~:
```javascript
addons 
  ByID: {}
  ByGuid: {}
  ...etc
  guides: [] // add here?
```

Also, I wasn't clear what isn't needed..

Reply:
don't load the addons in this reducer, use the existing `addons` reducer.

Reply:
okay I'll revisit this.. thanks!

Reply:
Sorry but I don't understand. You fetch addons from the search API. Instead of loading them in a `guides` reducer, you can use the existing `loadAddonResults()` action in your `guides` saga, no? That would load all the add-ons in the `addons` reducer.

You should not have to touch the addons reducer code.